pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/SegmentDeletionManager.java

```diff
@@ -207,8 +227,8 @@ protected void removeSegmentFromStore(String tableNameWithType, String segmentId
    * @param retentionInDays: retention for deleted segments in days
    */
   public void removeAgedDeletedSegments(int retentionInDays) {
-    if (_localDiskDir != null) {
-      File deletedDir = new File(_localDiskDir, DELETED_SEGMENTS);
+    if (_dataDir != null) {
+      File deletedDir = new File(_dataDir, DELETED_SEGMENTS);
```

Comment:
This should also be a pinotFs call, right?

Reply:
i will support remove aged segments in the next one, after my other PR for adding lastModified to PinotFS goes in